perf/stability/alertmanager/prometheusrule.yaml

```diff
@@ -17,16 +17,21 @@ spec:
           expr: >
             sum(irate(istio_requests_total{reporter="destination", response_code=~"5.*"}[5m])) / sum(irate(istio_requests_total{reporter="destination"}[5m])) > 0.05
           for: 5m
+        - alert: LatencyHigh
```

Comment:
We need to add an "expected 2xx" alarm.

Depending on the rps and number of namespaces, you can set it.

The following is the 200 reported by ingress - it excludes traffic to grafana, prom etc.

```
sum(rate(istio_requests_total{reporter="source", source_app="istio-ingressgateway",response_code="200",destination_service_namespace=~"service-graph.*"}[1m]))
```




sum(irate(istio_requests_total{reporter="destination", response_code=~"5.*"}[5m]))

Reply:
done